pyEpiabm/pyEpiabm/__init__.py

```diff
@@ -16,6 +16,7 @@
 from .microcell import Microcell  # noqa
 from .household import Household  # noqa
 from .parameters import Parameters  # noqa
+from .compartment_counter import CompartmentCounter  # noqa
```

Comment:
@NicholasFan235 can I run this change past you, had second thoughts that we might not want `CompartmentCounter` easily accessible if it's not meant to be called directly? Not really sure what I am talking about here - is this okay?

Reply:
Okay so will leave this but rename to `_CompartmentCounter`